### DISCUSSION TITLE
Nord Theme

### DISCUSSION THREAD

**Discussion by bfg-coding** [saadeghi/daisyui#1736] [Ideas / feature requests]
I could've sworn when I first came across Daisy UI it had the nord theme as one of the premade options.  Am I losing my mind or did something happen to it?   Would that be worth something supporting?

**Comment by saadeghi** [comment]
Maybe it was a dream 😅 there was never a built-in theme named "Nord"

**Comment by bfg-coding** [comment]
Yeah I guess I just assumed it was there since Night was there.  If I built it out would I be able to make a PR for it to be standard?

https://www.nordtheme.com/  here the one I'm referring to for those who look

**Reply by saadeghi** [reply]
I like this theme but:
1. I'm not sure if the name "Nord" is trademarked or not
2. I'm currently working on daisyUI v3 and there are possibly breaking changes in color values. I'm still not sure how it's going to be. It's a work in progress so I appreciate if you wait for the update before making a PR

**Reply by bfg-coding** [reply]
I will look that up for you and have no problem waiting until after v3 to work on a PR

**Reply by bfg-coding** [reply]
Chatted with the maintainer who saw this chat https://github.com/orgs/nordtheme/discussions/183#discussioncomment-5537101

**Reply by saadeghi** [reply]
Thank you. Sounds great 🙌

**Reply by bfg-coding** [reply]
Hey @saadeghi hope things are going good for you.   Do you by chance have an idea when you are releasing v3?

**Reply by saadeghi** [reply]
@bfg-coding Working on it. If everything goes right, it will be ready at the end of this month.

**Comment by saadeghi** [comment]
I found this today: https://github.com/HikaruEgashira/daisy-theme-nord

**Comment by ahsan-a** [comment]
any updates on this? would really like a nord dark mode for daisyui

**Reply by saadeghi** [reply]
We added Nord theme (light)  
There's not Nord Dark yet but you can extend it and change `base-*` colors to Nord dark color shades and it will be dark.
